lib/bin/lwm2m_carrier/lwm2m_carrier.rst

```diff
@@ -216,6 +219,13 @@ Below are some of the requirements and limitations of the application while runn
    * The LwM2M carrier library is currently only certified for the *LTE-M* LTE mode.
    * The :option:`CONFIG_LTE_NETWORK_USE_FALLBACK` should be disabled in your application, as seen in the :ref:`lwm2m_carrier` sample project configuration (:file:`nrf/samples/nrf9160/lwm2m_carrier/prj.conf`).
 
+* The LwM2M carrier library registers to receive several AT event reports using the :ref:`at_cmd_readme` and :ref:`at_notif_readme` libraries. The following notifications should not be deregistered by the application:
```

Comment:
It is maybe good to specify that if the customer or the application does not use the shared AT socket (or those modules), then they do not need to follow these rules.
AT notifications and events subscriptions are per AT socket clients.